app/Http/Controllers/SubGroupController.php

```
@@ -0,0 +1,99 @@
+            ]);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(SubGroupStoreRequest $request)
+    {
+        $validatedData = $request->validated();
+        $subGroup = DB::transaction(function () use ($validatedData) {
+            return SubGroup::create($validatedData);
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'store'. (undefined)

(StaticAccess)